api/k8s/application/k8sApplicationRestHandler.go

```diff
@@ -804,6 +805,21 @@ func (handler *K8sApplicationRestHandlerImpl) requestValidationAndRBAC(w http.Re
 	}
 }
 
+func (handler *K8sApplicationRestHandlerImpl) restrictTerminalAccessForNonSuperUsers(w http.ResponseWriter, token string) bool {
+	envVars := &bean2.TerminalEnvVariables{}
```

Comment:
move this to util/GlobalEnvVariables file and use from impl import

Reply:
done